src/controller/order-controller.ts

```diff
@@ -140,23 +213,99 @@ export async function captureOrderController(fastify: FastifyInstance) {
   });
 }
 
+// Patch order
+export type ShippingOption = {
+  id: string;
+  label: string;
+  type: string;
+  selected: boolean;
+  amount: {
+    value: string;
+    currency_code: string;
+  };
+};
+
+function calcShipping(address: ShippingAddress): String | boolean {
```

Comment:
I think we need to use the lower case string here to refer to the type. Also, I don't think this function will ever return a boolean.
```suggestion
function calcShipping(address: ShippingAddress): string {
```

Reply:
boolean `false` will reject the shipping address, i.e. if the merchant doesn't ship to there.  This can probably be explained better I just made it functional